mk/spksrc.tc-vers.mk

```diff
@@ -57,3 +57,7 @@ endif
 ifeq ($(TC_VERS),6.2.3)
 TC_BUILD = 25423
 endif
+
+ifeq ($(TC_VERS),7.0)
+TC_BUILD = 4000
```

Comment:
4000 (four-thousand) is definitively wrong, must be over fourthly-thousand.  I saw a build number of `41222` yesterday.

Reply:
Yes it is documented that for DSM 7 the min. os version is 7.0-40000 (but it was no problem to install adminer with  7.0-4000).

Reply:
Still, this needs to be readjusted as once we publish online it will require a proper build number.